Frontend/speedcart-react/src/components/TermsOfService/TermsOfService.js

```diff
@@ -1,10 +1,16 @@
 // src/components/TermsOfService.js
 import React, {useState} from 'react';
-import ReCAPTCHA from 'react-recaptcha';
+import Recaptcha from 'react-recaptcha';
 import inputStyles from '@modularStyles/inputs.module.css';
 import displayStyles from '@modularStyles/displayTypes.module.css';
+import { Link } from 'react-router-dom';
+import CustomCheckbox from '../CustomCheckbox';
```

Comment:
Change import path to use `@components` format instead and move these two imports above the CSS style imports.